queries.js

```diff
@@ -0,0 +1,43 @@
+const pgp = require('pg-promise')()
+const connectionString = 'postgres://localhost:5432/sweethornbillbookstore'
+const db = pgp(connectionString)
+
+
```

Comment:
oh, i made space to separate it (via the way my brain works) and i forgot to delete it. i took care of that!